src/app/space/create/deployments/resource-usage/loading-utilization-bar.component.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Component,
+  Input,
+  OnDestroy,
+  OnInit
+} from '@angular/core';
+
+export enum State {
+  Okay = 'utilization-okay',
+  Warning = 'utilization-warning'
+}
+
+@Component({
+  selector: 'loading-utilization-bar',
+  templateUrl: 'loading-utilization-bar.component.html',
+  styleUrls: ['./utilization-bar.component.less']
+})
+export class LoadingUtilizationBarComponent {
+
+  @Input() resourceTitle: string;
+  @Input() resourceUnit: string;
+
+  private currentState: State;
+
+  ngOnInit(): void {
+    this.currentState = State.Okay;
+  }
+
+  getUtilizationClass(): State {
+    return this.currentState;
+  }
```

Comment:
This component is completely static, so there's a bunch of machinery happening here to return the hardcoded 'utilization-okay' string to the [ngClass] in the template. Why not just inline it and get rid of all this component code?